README.md

```diff
@@ -212,6 +212,18 @@ For the "Debug Solution" to run properly on Windows 10/11 using WSL, the followi
 

```

Comment:
Can you add this section from the ODE repository here as well? I tried running this on a computer that didn't have this git config setup and it failed running the bash scripts.

#### Step 0 - For Windows Users Only

If running on Windows, please make sure that your global git config is set up to not convert end-of-line characters during checkout.

Disable `git core.autocrlf` (One Time Only)

```bash
git config --global core.autocrlf false
```

Reply:
Added